bitnami/redis-cluster/7.4/debian-12/prebuildfs/opt/bitnami/scripts/liblog.sh

```diff
@@ -39,7 +38,16 @@ stderr_print() {
 #   None
 #########################
 log() {
-    stderr_print "${CYAN}${MODULE:-} ${MAGENTA}$(date "+%T.%2N ")${RESET}${*}"
+    local prefix=""
+    local suffix=""
+    local color_bool="${BITNAMI_COLOR:-true}"
+    if [[ "$color_bool" = 1 || "$color_bool" =~ ^(yes|true)$ ]]; then
+        prefix="${CYAN}${MODULE:-} ${MAGENTA}$(date "+%T.%2N ")${RESET}"
+        suffix="${RESET}"
+    else
+        prefix="${MODULE:-} $(date "+%T.%2N ")"
+    fi
+    stderr_print "${prefix}${*}${suffix}"
```

Comment:
I think the `suffix` variable is not necessary here. We can simply use `${RESET}` in `stderr_print` and avoid allocating memory for it (even if it is almost free). Please consider this review for the rest of the functions with the same structure.